.travis_install.sh

```diff
@@ -6,7 +6,7 @@ bash miniconda.sh -b -p $HOME/miniconda
 export PATH="$HOME/miniconda/bin":$PATH
 conda config --set always_yes yes --set changeps1 no
 conda update -q conda
-conda install jupyter
+conda install jupyter typing
```

Comment:
Hmm, interesting that Travis' Python3 doesn't come with `typing` by default

Reply:
It's being aliased by miniconda, probably. I guess I could use pip to install jupyter, but I like conda better

Reply:
Or maybe not. It looks like my path isn't being updated...